src/builds.ts

```diff
@@ -98,6 +98,30 @@ export interface ProjectBuildOptions {
     /** Use babel to compile all ES6 JS down to ES5 for older browsers. */
     compile?: boolean
   };
+
+  /**
+   * Browser capabilities required for serving this build. Used by servers that
+   * support differential serving to decide which build to serve to a given
+   * user agent.
+   *
+   * Values include `es2015` and `push`. See canonical list at:
+   * https://github.com/Polymer/prpl-server-node/blob/master/src/capabilities.ts
+   */
+  capabilities?: string[];
```

Comment:
Sorry if this comment is coming late in the name-game for the prpl node server.

The property name feels a little awkward used like this.  `requiredCapabilities` is closer, but I sort of don't like that either.

A client block might make it clearer, like:
```js
client: {
   capabilities: ['es2015', 'push']
}
```

Alternately course maybe `clientFeatures` is a more conventional and intention revealing name than `capabilities` and it could be used on its own with more implication that they are required features of the client?